custom_components/elevenlabs_tts/tts.py

```diff
@@ -70,6 +83,10 @@ async def async_get_tts_audio(
         """Load TTS from the ElevenLabs API."""
         return await self._client.get_tts_audio(message, options)
 
+    def async_get_supported_voices(self, language: str) -> list[Voice] | None:
```

Comment:
ahh interesting, I tried getting this to work Sunday but the [docs](https://developers.home-assistant.io/docs/core/entity/tts/#get-supported-voices) said to return a list of strings...I guess wrong documentation wasn't helping me out haha.

Reply:
Alot of this integration code was inspired/copied from the Wyoming TTS integration :p